### DISCUSSION TITLE
플필수정시 온보딩으로 보내기(to.병준님)

### DISCUSSION THREAD

**Discussion by eunohhh** [nbc-final-a10/TripBuddy#270] [코드공유]
플필 수정시 온보딩의 특정 funnel 번호로 보내면 되게끔 하였습니다.
보내야하는 url과 내용을 정리합니다~!

### /onboarding?funnel=0&mode=edit
  - 닉네임 수정 
### /onboarding?funnel=2&mode=edit
  - 나이 수정(캘린더)
### /onboarding?funnel=3&mode=edit
  - 성별 수정
### /onboarding?funnel=4&mode=edit
  - 거주 지역 수정
### /onboarding?funnel=5&mode=edit
  - mbti 수정 
### /onboarding?funnel=7&mode=edit
  - 선호 버디 테마 수정
### /onboarding?funnel=8&mode=edit
  - 선호 여정 테마 수정
### /onboarding?funnel=9&mode=edit
  - 자기소개 수정

이상입니다! 그리고 현재 수정후 돌아가는 주소는 그냥 /profile/[id] 인 상태입니다!

**Comment by Captain-Kim** [comment]
확인했습니다. 오늘 중 작업하겠습니다. 감사합니다.

**Reply by Captain-Kim** [reply]
구현완료
